Simplified/NYPLProblemDocument.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+@objcMembers class NYPLProblemDocument: NSObject, Codable {
+  static let TypeNoActiveLoan =
+    "http://librarysimplified.org/terms/problem/no-active-loan";
+  static let TypeLoanAlreadyExists =
+    "http://librarysimplified.org/terms/problem/loan-already-exists";
+  
+  let type: String?
+  let title: String?
+  let status: Int?
+  let detail: String?
+  let instance: String?
+  
+  fileprivate init(_ dict: [String : Any]) {
+    self.type = dict["type"] as? String
+    self.title = dict["title"] as? String
+    self.status = dict["status"] as? Int
+    self.detail = dict["detail"] as? String
+    self.instance = dict["instance"] as? String
+    super.init()
+  }
+  
+  @objc static func fromData(_ data: Data) throws -> NYPLProblemDocument {
```

Comment:
Function explaining what this function does and maybe short one-liner about the parameters?